### PR TITLE
Add component schema for 2020 and 2018  versions

### PR DESCRIPTION
**Pull request by arolshan**

### CODE REVIEW

**Comment by yoramsinger** [comment]
@arolshan you need to add a new line at the end of one file.

File: schemas/common/autogeneratedResources.json [hunk 1]
```diff
@@ -2857,6 +2857,12 @@
         {
           "$ref": "https://schema.management.azure.com/schemas/2016-11-01/Microsoft.ImportExport.json#/resourceDefinitions/jobs"
         },
+        {
+          "$ref": "https://schema.management.azure.com/schemas/2018-05-01-preview/Microsoft.Insights.json#/resourceDefinitions/components"
```
Comment (anthony-c-martin) [inline schemas/common/autogeneratedResources.json:2861]: You shouldn't need to make any changes to this file - it's only for teams using the automatic generation pipeline. Please can you revert these changes?
